src/popup.css

```diff
@@ -55,3 +55,9 @@ header {
   background-color: var(--dark-gray);
   border-color: var(--dark-gray);
 }
+
+.refreshBtn {
```

Comment:
Let's think another way of positioning the refresh button instead of using a position style. For example, we could use flex to do the same behavior - I will leave the suggestion in popup.html :)

Reply:
doni